package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "json-ld.org",
+  "private": true,
+  "description": "json-ld.org homepage",
+  "scripts": {
+    "build": "npx @11ty/eleventy",
+    "serve": "npx @11ty/eleventy --serve",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "",
```

Comment:
We should switch this repo to the `BSD-3-Clause`. The W3C software and documentation license has increasingly fallen out of favor in place of ye olde (and far more common) BSD (thankfully).

Reply:
Removing the unused field.  This project uses package.json for build rules, not metadata.  There exists a LICENSE.md file for license details.